src/components/dialog/Dialog/DialogContentArea.tsx

```diff
@@ -5,25 +5,31 @@ import { ClassNameProps } from '../../types'
 
 export type DialogContentAreaProps = ClassNameProps & {
   children?: ReactNode
+  useScrollLayout?: boolean
 }
 
 export function DialogContentArea({
   children,
   className,
+  useScrollLayout = true,
```

Comment:
I would avoid prefixing a variable with `use` in React as it has a special meaning.

```suggestion
  enableScrollLayout = true,
```

Reply:
That's a good suggestion, thanks